superset-frontend/src/components/Select/Select.tsx

```diff
@@ -162,6 +170,16 @@ const StyledLoadingText = styled.div`
   `}
 `;
 
+const StyledHelperText = styled.div`
```

Comment:
This looks identical to the AsyncSelect one. Should we put in a common file and import it from there?

Reply:
Yes. I left it duplicated on purpose in this PR because @cccs-RyanK is already working on the common file in https://github.com/apache/superset/pull/21094